### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by chiedozie013**
In this project, I:
- created the form,
- used freespree for collecting data collection,
- checked linters for no errors
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,6 +223,52 @@ <h3>
...
+      <section>
+        <p class="contact-paragraph">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          id="contact-form"
+          action="https://formspree.io/f/xknlnqre"
+          method="post"
+        >
```
Comment (V-Blaze) [inline index.html]: > A user cannot submit the form if one or more of the fields is empty.
- - You have done great so far. :+1: :+1: But there are some requirements you still need to implement before moving to the next milestone. :+1: 
- [ ] As stated in the project requirement, all input fields needs to be filled before a user can submit a form. Currently that's not the case in your project. I'll suggest you add the `required` attribute to all the inputs in order to meet this requirement. :+1: :wave:

File: index.html [hunk 2]
```diff
@@ -223,6 +223,52 @@ <h3>
...
+          />
+          <input type="email" name="email" placeholder="Email address" />
+          <textarea
+            name="message"
+            id="message"
+            type="text"
+            maxlength="500"
+            placeholder="Write me something..."
+          ></textarea>
+          <button class="formbutton" type="submit">Get in touch</button>
```
Comment (V-Blaze) [inline index.html]: > Implement the button interactions (enable, hover, etc.).

- [ ] The button interactions like the hover is not implemented in this submit button as stated in the project requirement. I'll suggest you look at the Figma designs dn get the text and background color for different button states. :wave: 

- - FIGMA:
Screenshot from 2023-07-11 13-50-12

- [ ] Also I'll suggest you adjust the height of this button to fully match the Figma design as your button height is a bit smaller than the given design: :+1: :+1:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @chiedozie013  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Added form spree action✔
-  Correct Github flow👋 
-  Linters setup correctly ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @chiedozie013, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**